### PR TITLE
ASA-278: Convert to pyproject.toml

### PR DESCRIPTION
**Pull request by bgrnwd**
### Description

Converting from setup.py to pyproject.toml. Also mocked out an API call in one of the unit tests

### Checklist

- [x] Code compiles correctly
- [x] Created tests which fail without the change (if applicable)
- [x] All lint and unit tests passing
- [x] Extended the README / documentation, if necessary

### Additional Comments

Anything else you'd like to add.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11"
+]
+dependencies = [
+    "requests==2.31.0",
+    "CacheControl==0.12.11",
+    "rapidfuzz==3.0.0",
+    "pandas==2.0.1"
```
Comment (tyler-richardett) [inline pyproject.toml:30]: Looks like you went backward on the versions of `CacheControl` and `pandas`. Not sure if that was intentional.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+[project]
+name = "itscalledsoccer"
+description = "A Python wrapper around the unofficial FotMob API"
```
Comment (tyler-richardett) [inline pyproject.toml:3]: Lol

**Review by tyler-richardett — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (bgrnwd) [reply pyproject.toml:30]: Nope, good catch

File: pyproject.toml (hunk 2 above)
Comment (bgrnwd) [reply pyproject.toml:3]: Oops

**Review by tyler-richardett — APPROVED** [review]
lgtm

**Pull request merged**